clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/controllers/KubernetesLoadBalancerController.groovy

```diff
@@ -19,21 +19,27 @@ package com.netflix.spinnaker.clouddriver.kubernetes.controllers
 import com.netflix.spinnaker.cats.cache.Cache
 import com.netflix.spinnaker.clouddriver.kubernetes.cache.Keys
 import com.netflix.spinnaker.clouddriver.kubernetes.model.KubernetesLoadBalancer
+import com.netflix.spinnaker.clouddriver.model.LoadBalancerProviderTempShim
 import org.springframework.beans.factory.annotation.Autowired
 import org.springframework.web.bind.annotation.RequestMapping
 import org.springframework.web.bind.annotation.RequestMethod
 import org.springframework.web.bind.annotation.RestController
 
+import javax.naming.OperationNotSupportedException
+
 @RestController
 @RequestMapping("/kubernetes/loadBalancers")
-class KubernetesLoadBalancerController {
+class KubernetesLoadBalancerController implements LoadBalancerProviderTempShim {
   private final Cache cacheView
 
   @Autowired
   KubernetesLoadBalancerController(Cache cacheView) {
     this.cacheView = cacheView
   }
 
+  // TODO(lwander): Groovy allows this to compile just fine, even though KubernetesLoadBalancer does
```

Comment:
At this point, nothing, though when the controllers come together there may be work to massage the model class. If not, I'll remove the TODO.